.appveyor.yml

```diff
@@ -8,7 +12,7 @@ install:
 build: false
 
 test_script:
-  - cargo test --locked -- --test-threads 1
+  - cargo test --release --locked -- --test-threads 1
```

Comment:
i think we can also remove the test-threads req here now

Reply:
should we add `--release` to `.travis.yml` as well? https://github.com/rustwasm/wasm-pack/blob/master/.travis.yml#L39

Reply:
Nah that's fine to be non-release as it's not producing a final binary. AppVeyor right now doesn't have separate jobs (like Travis does) for building releases and testing, and from my experience at least that tends to work out alright.

In that sense this is just needed here for AppVeyor because we're both testing a binary and publishing it, whereas Travis either publishes it or tests it. Testing without `--release` is generally preferred for Travis because it gets you results more quickly

Reply:
Also certainly, can remove `--test-threads`!